src/wedderburn.jl

```diff
@@ -51,9 +51,9 @@ function subset_permutation(
     g::Groups.GroupElement,
     act::AlphabetPermutation
 )
-    subset_idies = Dict(subset[i] => i for i in Int32.(eachindex(subset)))
+    subset_idies = Dict(subset[i] => i for i in UInt32.(eachindex(subset)))
```

Comment:
`UInt32.(eachindex(subset))` materializes the whole vector which you definitely don't want to...